cypress/integration/page_size_selector.ts

```diff
@@ -1,104 +0,0 @@
-// / <reference types="Cypress" />
```

Comment:
Removed this test since the PageSizeSelector is effectively being tested in several other tests this seems fairly redundant.